doc/plutus/explanations/plutus-tools-component-descriptions.rst

```diff
@@ -0,0 +1,321 @@
++--------------+---------------------------------------------------------------+
+| State        | Maintained but not extended. Unstable interface.              | 
++--------------+---------------------------------------------------------------+
+
+8. Contract monad emulator
+-----------------------------------------------------------
+
+`Contract monad emulator <https://github.com/input-output-hk/plutus-apps/tree/main/plutus-contract>`_ 
+is a library that provides an environment for emulating the blockchain. The environment 
+provides a way for writing traces for the contract which are sequences of actions 
+by simulated wallets that use the contract. The component is highly dependent on the 
+Contract Monad. However, there are plans to decouple it from the Contract Monad 
+so that it can be used with other libraries.
+
++--------------+---------------------------------------------------------------+
+| Contract monad emulator                                                      |
++==============+===============================================================+
+| Location     | ``plutus-apps/plutus-contract``                               |
++--------------+---------------------------------------------------------------+
+| Dependencies | cardano-api, plutus-ledger-api, plutus-ledger,                |
+|              | plutus-ledger-constraints, plutus-chain-index-core,           |
+|              | plutus-contract                                               |
++--------------+---------------------------------------------------------------+
+| State        | Maintained and will be extended. Unstable interface.          | 
++--------------+---------------------------------------------------------------+
+
+9. Plutus contract state machine
+-----------------------------------------------------------
+
+`Plutus contract state machine <https://github.com/input-output-hk/plutus-apps/tree/main/plutus-contract>`_ 
+is a library that provides a high-level model for writing a Plutus application based 
+on the State Machine formalism. Even though it is inefficient, it can be used to 
+model a Plutus application and to test the actual implementation.
+
++--------------+---------------------------------------------------------------+
+| Plutus contract state machine                                                |
++==============+===============================================================+
+| Location     | ``plutus-apps/plutus-contract``                               |
++--------------+---------------------------------------------------------------+
+| Dependencies | plutus-contract                                               |
++--------------+---------------------------------------------------------------+
+| State        | We maintain it, but it is not production ready. It is         |
+|              | inefficient---it creates Plutus applications that have high   |
```

Comment:
```suggestion
|              | inefficient---it creates Plutus applications that have too high   |
```

Reply:
@sjoerdvisscher I think you're making an important point about how we are describing the tools. I agree that we need to really consider how our descriptions may impact others' perceptions of the tools. Who are we targeting as our audience here? How do we want to impact them?

Reply:
Why would we want to talk about items for which we say "don't use it?" Why have them in the repo? Would it make more sense to get rid of it so people don't find it?